Modules/Microsoft365DSC/DSCResources/MSFT_EXOQuarantinePolicy/readme.md

```diff
@@ -0,0 +1,5 @@
+# EXOQuarantinePolicy
+
+## Description
+
+Create or modify a new EXOQuarantinePolicy in your cloud-based organization.
```

Comment:
I would go with "Create or modify a EXOQuarantinePolicy in your cloud-based organization." (remove the 'new')

Reply:
fixed